server/sonar-server/src/main/java/org/sonar/server/rule/index/RuleDoc.java

```diff
@@ -197,4 +200,35 @@ public String toString() {
     return ReflectionToStringBuilder.toString(this);
   }
 
+  public static RuleDoc of(RuleForIndexingDto ruleForIndexingDto) {
+    RuleDoc ruleDoc = new RuleDoc()
+      .setKey(ruleForIndexingDto.getRuleKey().toString())
+      .setRepository(ruleForIndexingDto.getRepository())
+      .setInternalKey(ruleForIndexingDto.getInternalKey())
+      .setIsTemplate(ruleForIndexingDto.isTemplate())
+      .setLanguage(ruleForIndexingDto.getLanguage())
+      .setName(ruleForIndexingDto.getName())
+      .setRuleKey(ruleForIndexingDto.getPluginRuleKey())
+      .setSeverity(ruleForIndexingDto.getSeverityAsString())
+      .setStatus(ruleForIndexingDto.getStatus().toString())
+      .setType(ruleForIndexingDto.getTypeAsRuleType())
+      .setCreatedAt(ruleForIndexingDto.getCreatedAt())
+      .setUpdatedAt(ruleForIndexingDto.getUpdatedAt());
+
+    if (ruleForIndexingDto.getPluginRuleKey() != null && ruleForIndexingDto.getRepository() != null) {
+      ruleDoc.setTemplateKey(RuleKey.of(ruleForIndexingDto.getPluginRuleKey(), ruleForIndexingDto.getRepository()).toString());
+    } else {
+      ruleDoc.setTemplateKey(null);
+    }
+
+    if (ruleForIndexingDto.getDescription() != null && ruleForIndexingDto.getDescriptionFormat() != null) {
+      if (RuleDto.Format.HTML == ruleForIndexingDto.getDescriptionFormat()) {
+        ruleDoc.setHtmlDescription(ruleForIndexingDto.getDescription());
+      } else {
+        ruleDoc.setHtmlDescription(Markdown.convertToHtml(ruleForIndexingDto.getDescription()));;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)